Source: Timo Gurr <tgurr@exherbo.org>
Upstream: No
Reason: Run into (endless) loops with sydbox and we added xfails for most them already

diff -Naur glibc-2.28/resolv/Makefile glibc-2.28.new/resolv/Makefile
--- glibc-2.28/resolv/Makefile	2018-08-01 07:10:47.000000000 +0200
+++ glibc-2.28.new/resolv/Makefile	2018-08-01 17:53:39.487847441 +0200
@@ -42,28 +42,15 @@
 routines += gai_sigqueue
 
 tests += \
-  tst-bug18665 \
-  tst-bug18665-tcp \
   tst-ns_name \
   tst-ns_name_compress \
   tst-ns_name_pton \
   tst-res_hconf_reorder \
   tst-res_hnok \
-  tst-res_use_inet6 \
-  tst-resolv-basic \
-  tst-resolv-binary \
-  tst-resolv-edns \
-  tst-resolv-network \
   tst-resolv-res_init-multi \
-  tst-resolv-search \
 
 # These tests need libdl.
 ifeq (yes,$(build-shared))
-tests += \
-  tst-resolv-ai_idn \
-  tst-resolv-ai_idn-latin1 \
-  tst-resolv-ai_idn-nolibidn2 \
-  tst-resolv-canonname \
 
 # uses DEPRECATED_RES_USE_INET6 from <resolv-internal.h>.
 tests-internal += \
@@ -168,23 +155,6 @@
 	  $(objpfx)tst-resolv-res_ninit.mtrace > $@; \
 	  $(evaluate-test)
 
-$(objpfx)tst-bug18665-tcp: $(objpfx)libresolv.so $(shared-thread-library)
-$(objpfx)tst-bug18665: $(objpfx)libresolv.so $(shared-thread-library)
-$(objpfx)tst-res_use_inet6: $(objpfx)libresolv.so $(shared-thread-library)
-$(objpfx)tst-resolv-ai_idn: \
-  $(libdl) $(objpfx)libresolv.so $(shared-thread-library)
-$(objpfx)tst-resolv-ai_idn-latin1: \
-  $(libdl) $(objpfx)libresolv.so $(shared-thread-library)
-$(objpfx)tst-resolv-ai_idn-nolibidn2: \
-  $(libdl) $(objpfx)libresolv.so $(shared-thread-library)
-$(objpfx)tst-resolv-ai_idn.out: $(gen-locales)
-$(objpfx)tst-resolv-ai_idn-latin1.out: $(gen-locales)
-$(objpfx)tst-resolv-ai_idn-nolibidn2.out: \
-  $(gen-locales) $(objpfx)tst-no-libidn2.so
-$(objpfx)tst-resolv-basic: $(objpfx)libresolv.so $(shared-thread-library)
-$(objpfx)tst-resolv-binary: $(objpfx)libresolv.so $(shared-thread-library)
-$(objpfx)tst-resolv-edns: $(objpfx)libresolv.so $(shared-thread-library)
-$(objpfx)tst-resolv-network: $(objpfx)libresolv.so $(shared-thread-library)
 $(objpfx)tst-resolv-res_init: $(libdl) $(objpfx)libresolv.so
 $(objpfx)tst-resolv-res_init-multi: $(objpfx)libresolv.so \
   $(shared-thread-library)
@@ -192,11 +162,8 @@
   $(shared-thread-library)
 $(objpfx)tst-resolv-qtypes: $(objpfx)libresolv.so $(shared-thread-library)
 $(objpfx)tst-resolv-rotate: $(objpfx)libresolv.so $(shared-thread-library)
-$(objpfx)tst-resolv-search: $(objpfx)libresolv.so $(shared-thread-library)
 $(objpfx)tst-resolv-threads: \
   $(libdl) $(objpfx)libresolv.so $(shared-thread-library)
-$(objpfx)tst-resolv-canonname: \
-  $(libdl) $(objpfx)libresolv.so $(shared-thread-library)
 
 $(objpfx)tst-ns_name: $(objpfx)libresolv.so
 $(objpfx)tst-ns_name.out: tst-ns_name.data
